optimade/grammar/v0.10.0.g

```diff
@@ -0,0 +1,63 @@
+// optimade v0.9.7 grammar spec in lark grammar format
```

Comment:
There is already an open PR #52 for v.0.10 of the grammar, would you mind coordinating with that instead so the appropriate people can review it there before merging this?

Reply:
I marked my v.0.10.0.g as v.0.10.0.elastic.g and added a comment to it. This should make clear that this grammar is supposed to work with the elastic filtertransformer until, we have a better solution to deal with transformer->grammar dependencies.